tests/testthat/test_do_adjustRtime-functions.R

```diff
@@ -1,9 +1,9 @@
 test_that("getPeakGroupsRtMatrix works", {
     param <- PeakGroupsParam()
     nSamples <- length(fileNames(xod_xg))
-    pkGrp <- .getPeakGroupsRtMatrix(
+    pkGrp <- xcms:::.getPeakGroupsRtMatrix(
```

Comment:
So all non-exported functions in tests need to be prefixed ?

Reply:
No - that's a leftover when I run the tests line by line locally.